README.md

```diff
@@ -7,28 +7,73 @@ Orange3 Text
 
 Orange3 Text extends [Orange3](http://orange.biolab.si), a data mining software
 package, with common functionality for text mining. It provides access
-to publicly available data, like NY Times, Twitter and PubMed. Further,
+to publicly available data, like NY Times, Twitter, Wikipedia and PubMed. Furthermore,
 it provides tools for preprocessing, constructing vector spaces (like
-bag-of-words, topic modeling and word2vec) and visualizations like word cloud
+bag-of-words, topic modeling, and similarity hashing) and visualizations like word cloud
 end geo map. All features can be combined with powerful data mining techniques
 from the Orange data mining framework.
 
-Installation
-------------
+Anaconda installation
+---------------------
 
-To install the add-on with pip use
+The easiest way to install Orange3-Text is with Anaconda distribution. Download [Anaconda](https://www.continuum.io/downloads) 
+for your OS (Python version 3.5). In your Anaconda Prompt first add conda-forge to your channels:
+
+    conda config --add channels conda-forge
+
+Then install Orange3
+
+    conda install orange3
+
+This will install the latest release of Orange. Install biopython to access PubMed widget (optional!)
+
+    conda install biopython
+
+Biopython is an optional dependency and not installing it will simply result in a missing PubMed widget. Then
 
     pip install Orange3-Text
 
-To install the add-on from source, run
+Run
+
+   orange-canvas
```

Comment:
on OSX, orange-canvas (created by conda) runs Python.app (instead of the python executable), and the former does not have qt.conf in correct location (see the link above)